src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -1504,13 +1505,18 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         new ItemStack[] {SlimefunItems.SILVER_INGOT, SlimefunItems.DAMASCUS_STEEL_INGOT, SlimefunItems.SILVER_INGOT, SlimefunItems.DAMASCUS_STEEL_INGOT, SlimefunItems.ELECTRIC_MOTOR, SlimefunItems.DAMASCUS_STEEL_INGOT, SlimefunItems.SILVER_INGOT, SlimefunItems.DAMASCUS_STEEL_INGOT, SlimefunItems.SILVER_INGOT})
         .register(plugin);
 
+        new EnergyConnector(categories.electricity, SlimefunItems.ENERGY_CONNECTOR, RecipeType.ENHANCED_CRAFTING_TABLE,
+        new ItemStack[] {SlimefunItems.CARBON, SlimefunItems.CARBON, SlimefunItems.CARBON, SlimefunItems.CARBON, SlimefunItems.COPPER_INGOT, SlimefunItems.CARBON, SlimefunItems.CARBON, SlimefunItems.CARBON, SlimefunItems.CARBON},
```

Comment:
ugh, I didn't even really see the recipe. That's a lot of CARBON...

How about this?
```
CWC
WRW
CWC
````
C = Carbon
W = Copper WIRE
R = Redstone block

Reply:
Well... Insulation lol

Reply:
how about 4 carbon (empty corners)?
And honestly I dont think many people would make this cuz you can just put another capacitator there - makes it less clustered overall